manifests/a/AbdelilahElAissaoui/Gitnuro/1.5.0/AbdelilahElAissaoui.Gitnuro.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: AbdelilahElAissaoui.Gitnuro

```

```suggestion
PackageIdentifier: JetpackDuba.Gitnuro
```
Because this comes from a GitHub release, it is preferred to use the GitHub repository as the identifier